docker/Dockerfile

```diff
@@ -10,6 +10,10 @@
 ### The date in the Server command must either match, or be more recent than,
 ### the date in the FROM command
 ### Date format is ISO year/month/day
+
+
+
+
```

Comment:
Unrelated change

Reply:
(Context, given that this is your first PR)

We try to avoid making changes to files incidentally, keeping PRs as to-the-point and relevant to their objective as possible. This makes it easier to go back and understand how and when files have changed in the past, because you don't have to look at unrelated, inconsequential changes, evaluating whether they are relevant.